apps/dapp-console/app/faucet/components/AuthenticatedHeader.tsx

```diff
@@ -0,0 +1,75 @@
+import { Authentications } from '@/app/faucet/types'
+import { RiCheckboxCircleFill } from '@remixicon/react'
+import { Text } from '@eth-optimism/ui-components/src/components/ui/text/text'
+import Image from 'next/image'
+import { useMemo } from 'react'
+
+type Props = {
+  authentications: Authentications
+}
+
+const getAuthenticatedLabelAndImages = (authentications: Authentications) => {
+  // Define the labels and images for each authentication type
+  const data: Record<keyof Authentications, { label: string; image: string }> =
+    {
+      coinbase: { label: 'Coinbase', image: '/logos/coinbase-logo.png' },
+      worldId: { label: 'World ID', image: '/logos/worldid-logo.png' },
+      gitcoin: { label: 'Gitcoin', image: '/logos/gitcoin-logo.png' },
+      eas: { label: 'EAS', image: '/logos/eas-logo.png' },
+    }
```

Comment:
supernit: can be defined outside of the function. will make function shorter and slightly easier to grok